db/seeds.rb

```diff
@@ -7,17 +7,19 @@
 #   Character.create(name: 'Luke', movie: movies.first)
 
 1.upto(5) do |i|
-	Employee.where(email: "employee#{i}@test.com").first_or_create!(password:"password")
+  Employee.where(email: "employee#{i}@test.com").first_or_create!(password: 'password')
 end
 puts 'Admin account creation'
-Admin.where(email: "admin@test.com").first_or_create!(password:"admintest")
-
+Admin.where(email: 'admin@test.com').first_or_create!(password: 'admintest')
 
+puts 'Company values creation'
+CompanyValue.where(title: 'Honesty').first_or_create!
+CompanyValue.where(title: 'Ownership').first_or_create!
+CompanyValue.where(title: 'Accountability').first_or_create!
+CompanyValue.where(title: 'Passion').first_or_create!
 2.upto(5) do |i|
-Kudo.where(title:" Seeds for employee#{i}@test.com").first_or_create!(content: "smh",giver: Employee.find_by(email: 'employee1@test.com'), receiver: Employee.find_by(email: "employee#{i}@test.com"))
+  Kudo.where(title: " Seeds for employee#{i}@test.com").first_or_create!(content: 'smh',
+                                                                         giver: Employee.find_by(email: 'employee1@test.com'),
```

Comment:
consider using `sample` method
```suggestion
                                                                         giver: Employee.all.sample,
```